content/en/docs/ops/ambient/getting-started/snips.sh

```diff
@@ -308,20 +324,36 @@ kubectl exec deploy/sleep -- sh -c "for i in \$(seq 1 100); do curl -s http://$G
 }
 
 snip_uninstall_1() {
-istioctl x waypoint delete --all
-istioctl uninstall -y --purge
-kubectl delete namespace istio-system
+kubectl label namespace default istio.io/dataplane-mode-
 }
 
 snip_uninstall_2() {
-kubectl label namespace default istio.io/dataplane-mode-
-}
+kubectl logs ds/ztunnel -n istio-system  | grep inpod
+}
+
+! IFS=$'\n' read -r -d '' snip_uninstall_2_out <<\ENDSNIP
+Found 3 pods, using pod/ztunnel-jrxln
+inpod_enabled: true
+inpod_uds: /var/run/ztunnel/ztunnel.sock
+inpod_port_reuse: true
+inpod_mark: 1337
+2024-03-26T00:02:06.161802Z  INFO ztunnel::inpod::workloadmanager: handling new stream
+2024-03-26T00:02:06.162099Z  INFO ztunnel::inpod::statemanager: pod received snapshot sent
+2024-03-26T00:41:05.518194Z  INFO ztunnel::inpod::statemanager: pod WorkloadUid("7ef61e18-725a-4726-84fa-05fc2a440879") received netns, starting proxy
+2024-03-26T00:50:14.856284Z  INFO ztunnel::inpod::statemanager: pod delete request, draining proxy
+ENDSNIP
 
 snip_uninstall_3() {
+istioctl x waypoint delete --all
+istioctl uninstall -y --purge
+kubectl delete namespace istio-system
+}
+
+snip_uninstall_4() {
 kubectl delete -f samples/sleep/sleep.yaml
 kubectl delete -f samples/sleep/notsleep.yaml
 }
 
-snip_uninstall_4() {
+snip_uninstall_5() {
```

Comment:
You need to add a call to snip_uninstall_5 in the @cleanup section of both test.sh and gtwapi_test.sh

Reply:
I added the uninstall calls, thanks